sequel/lib/utils.js

```diff
@@ -30,6 +30,25 @@ utils.object.hasOwnProperty = function(obj, prop) {
 
 
 /**
+ * String literal helper function
+ *
+ * Pass in a value and have it wrapped in a string literal.
+ * Saves having self.escapeCharacter littered everywhere
+ *
+ * @param {String} val
+ * @param {String} escapeCharacter
+ * @return {String}
+ * @api public
+ *
+ */
+
+utils.stringLiteral = function stringLiteral(val, escapeCharacter) {
+  var regex = new RegExp(escapeCharacter, 'g');
+  return escapeCharacter+val.replace(regex, '\\'+escapeCharacter)+escapeCharacter;
```

Comment:
What other approach do you recommend? I don't see anything wrong with it personally. I welcome your thoughts though.